schematic/store/synapse.py

```diff
@@ -1343,6 +1369,18 @@ def add_entities(
             manifest (pd.DataFrame): modified to add entitiyId as appropriate.
 
         '''
+
+        # Expected behavior is to annotate files if `Filename` is present regardless of `-mrt` setting
+        if 'filename' in [col.lower() for col in manifest.columns]:
+            # get current list of files and store as dataframe
+            dataset_files = self.getFilesInStorageDataset(datasetId)
```

Comment:
can you update the variable naming here to make the differences more clear, to me its not clear upon reading how `files` differs exactly from `dataset_files`, since it takes `dataset_files` as a parameter, furthermore `files` still contains information for the entire dataset.

Maybe something along the lines of: `filenames_list` and `file_info_dict`

Reply:
Ahh I see how that could be confusing, I'll update